packages/lodestar/src/eth1/provider/jsonRpcHttpClient.ts

```diff
@@ -121,10 +127,20 @@ export class JsonRpcHttpClient implements IJsonRpcHttpClient {
     }
 
     try {
+      let headers;
```

Comment:
```ts
const headers = {"Content-Type": "application/json"}
```
then if condition mutate `headers`

Reply:
:+1: , done in https://github.com/ChainSafe/lodestar/pull/3777/commits/8f8197d55d2021f04fa1927cd6bd76c9b4db1c5d